chainer_compiler/chainer_compiler.py

```diff
@@ -203,6 +203,34 @@ def _run_translator(translator, mc, inputs):
     return graph
 
 
+def _resolve_name_correspondence(mc, params, name, param_values):
```

Comment:
How about `_resolve_name_correspondence_bn`? This logic is very specific for BN.